### PR TITLE
fix(chart): use single color on bar and bubble chart

### PR DESCRIPTION
**Pull request by Sarin-Udompanish**
## Description
When the dataset is single, the bar and bubble charts display incorrect background colors

Step to reproduce
1. Open https://codesandbox.io/s/bar-chart-with-a-single-dataset-k67zkf

Expected Results
All the bars should have the same colors.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/chart/elements/chart.ts [hunk 1]
```diff
@@ -402,11 +402,7 @@ export class Chart extends BasicElement {
         // These types, Colors could be string or array
```
Comment (wsuwt) [inline packages/elements/src/chart/elements/chart.ts]: Is this comment still relevant to the new change ?

Comment (Sarin-Udompanish) [reply packages/elements/src/chart/elements/chart.ts]: updated

**Review by wsuwt — APPROVED** [review]
